kemdk/src/main/java/dk/gls/kemdk/emdk/EMDKThrowable.kt

```diff
@@ -0,0 +1,37 @@
+package dk.gls.kemdk.emdk
+
+import dk.gls.kemdk.model.EMDKConfigResult
+import dk.gls.kemdk.oemInfo.OEMInfoThrowable
+
+sealed class EMDKThrowable(override val cause: Throwable?) : Throwable() {
+    /**
+     * ProfileLoadThrowable is thrown when there are problems loading the emdk manager
+     * The EMDK manager might be null on startup
+     */
+    class ProfileLoadThrowable(cause: Throwable) : EMDKThrowable(cause)
+
+    /**
+     * ProfileXMLThrowable is related to errors in the EDMKConfig.xml
+     * See Readme for troubleshooting
+     */
+    class ProfileXMLThrowable(val emdkConfigResult: EMDKConfigResult?) : EMDKThrowable(IllegalStateException("Configure the XML profile accordingly to which data you want to retrieve"))
```

Comment:
Regarding comment about check_xml error vs success -> I think this throwable is if xml is misconfigured/bad

Reply:
This is how we have had it implemented all the time. I have pulled it from these examples originally. 
https://techdocs.zebra.com/emdk-for-android/6-3/guide/xmlresponseguide/

I have corrected the if statement in EMDKListener so that it is now emits error when != SUCCESS

Reply:
'In most of the cases, the response Status Code will be [CHECK_XML](https://techdocs.zebra.com/emdk-for-android/6-3/api/core/EMDKResults-STATUS_CODE/) regardless of whether the profile was successfully applied or not.'